css/app.css

```diff
@@ -1,38 +1,48 @@
 @import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@500&display=swap');
 
-body {
-  font-family: 'Work Sans';
-  font-style: normal;
-  font-weight: 600;
-  font-size: 30px;
-  line-height: 35px;
-  letter-spacing: 0.1em;
-
-  color: #000000;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
   background: #ffe4c2;
-  text-align: center;
+  font-family: 'Work Sans';
+  outline: none;
 }
 
-/* default theme is set to white theme */
+/* ========================== */
+/*         Toggle Icon        */
+/* ========================== */
+
+#icon {
+  float: right;
+  margin-right: 50px;
+  padding-top: 20px;
+  width: 50px;
+  height: 60px;
+  cursor: pointer;
+}
 
 :root {
   --primary-color: #ffffff;
-  --presult-color: #e1efff;
-  --phistory-color: #272727;
-  --poutput-color: #747474;
-  --pclear-color: #00dec3;
-  --pnumber-color: #e1efff;
-  --ptext-color: #000000;
-  --poperator-color: #89c0ff;
-
   --secondary-color: #0d0d0d;
+
+  --presult-color: #e1efff;
   --sresult-color: #1c1c1c;
```

Comment:
Do we really require these secondary colors? Can you try removing them? Based on what I can see they are not required and should be removed. Let me know your thoughts here.